app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :user_id, dependent: :destroy
+  has_many :likes, foreign_key: :user_id, dependent: :destroy
+
```

Comment:
- [ ] Great work adding this association as instructed by the previous reviewer. However, as per the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) presented to you, the foreign key should be `author_id` not `user_id`. So just like you have done in your posts association, you need to do the same to the comments and likes association.

Reply:
Hi @peterrobert, please in the ERD diagram **author_id** is only for posts and they specify to remember that case.
"
- Foreign keys should be included.
  - Pay attention that in posts table the column holding the foreign key to the users table should be named author_id "
  

https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md

Reply:
@touredri , No if you check the ERD diagram even the like and the comment had author_id instead of user_id. Kindly check below ⬇️ 
![Screenshot 2023-07-15 at 11 24 38](https://github.com/touredri/blog/assets/32361963/2d8907df-96b6-40ed-b815-8a0398d8cf6a)

Reply:
ok thx

Reply:
well resolved! ✔️